packages/docs/.vuepress/components/IncludeContent.vue

```diff
@@ -0,0 +1,9 @@
+<template>
+<Content :page-key="$site.pages.find(p => p.path === '/' + contentKey + '.html').key" />
```

Comment:
File is not new, only path has changed from `p.path === '/core/'` to `p.path === '/'`